templates/robotics/ros-esm.html

```diff
@@ -1,273 +1,323 @@
+      </p>
+      <p>
+        With ROS ESM, keeping core ROS packages secure is a fast and
+        straightforward process.
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section--shallow">
+  <hr class="p-rule is-fixed-width" />
+  <div class="p-section--shallow u-fixed-width">
+    <h2>ROS ESM security coverage</h2>
+  </div>
+  <div class="row">
+    <div class="col-3"></div>
+    <div class="col-9">
+      <hr class="p-rule--muted" />
+      <ul class="p-list--divided">
+        <li class="p-list__item">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">Main</h3>
+            </div>
+            <div class="col-6">
+              <p>
+                2,300 packages in the Ubuntu Main repo supported for 5 years
+                after EOL
+              </p>
             </div>
           </div>
-          <div class="p-tiered-card--medium">
-            <div class="row">
-              <div class="col-2">
-                <h3 class="p-heading--2">Universe</h3>
-              </div>
-              <div class="col-4">
-                <p class="p-card__content">An additional 23,000+ packages in the Ubuntu Universe repository for 10 years.</p>
-              </div>
+        </li>
+        <li class="p-list__item">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">Universe</h3>
+            </div>
+            <div class="col-6">
+              <p>
+                An additional 23,000+ packages in the Ubuntu Universe repository
+                for 5 years after EOL
+              </p>
             </div>
           </div>
-          <div class="p-tiered-card--low">
-            <div class="row">
-              <div class="col-2">
-                <h3 class="p-heading--2">Main</h3>
-              </div>
-              <div class="col-4">
-                <p class="p-card__content">2,300 packages in the Ubuntu Main repo supported for 5 years after EOL.</p>
-              </div>
+        </li>
+        <li class="p-list__item">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">Security</h3>
+            </div>
+            <div class="col-6">
+              <p>
+                2,300 packages in the Ubuntu Main repo supported for 5 years
+                after EOL
+              </p>
             </div>
           </div>
-        </div>
-      </div>
+        </li>
+      </ul>
     </div>
-    <hr class="p-rule is-fixed-width" />
-  </section>
-  <section class="p-strip u-no-padding--top u-no-padding--bottom">
-    <div class="row">
-      <div class="col-8">
-        <h2>ROS distributions available today in ROS ESM:</h2>
-      </div>
+  </div>
+</section>
+
+<section class="p-section--shallow">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row">
+    <div class="col-8 p-section--shallow">
+      <h2>ROS distributions available today in ROS ESM:</h2>
     </div>
```

Comment:
```suggestion
    <div class="col-8">
        <div class="p-section--shallow">
            <h2>ROS distributions available today in ROS ESM:</h2>
        </div>
    </div>
```
Better practice to separate spacing elements from grid elements